wa/framework/uiauto/app/build.gradle

```diff
@@ -2,7 +2,8 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 28
-    buildToolsVersion '28.0.3'
+    // buildToolsVersion is no longer used/accepted.
+    // buildToolsVersion '28.0.3'
```

Comment:
I think this change is actually spurious, as I didn't intend to modify the original library. But here's some more input...

The original android.support uiauto library should still build and work fine. If we use a new enough gradle it will complain about some deprecated bits (like buildToolsVersion), but this is fixable.

My general idea while putting this together was to first have an androidx version of our uiauto library (kept in a separate directory) that worked with jetnews. Once that is known to work, we could start migrating the other workloads to use the new androidx library.

Also, the UIObject class (and friends) are eventually going to be deprecated in favor of UIObject2. I think these potential changes will be more invasive, as UIObject2 doesn't provide the equivalent functionality of UIObject / UIScrollable etc.